tools/server-spring/src/main/java/org/eclipse/rdf4j/http/server/ProtocolExceptionResolver.java

```diff
@@ -78,15 +78,14 @@ public ModelAndView resolveException(HttpServletRequest request, HttpServletResp
 
 			StringWriter stringWriter = new StringWriter();
 
-			// We choose NQUADS because we want to support streaming in the future, and because there could be a use for
-			// different graphs in the future
-			Rio.write(validationReportModel, stringWriter, RDFFormat.NQUADS);
+			// We choose RDFJSON because this format doesn't rename blank nodes.
+			Rio.write(validationReportModel, stringWriter, RDFFormat.RDFJSON);
```

Comment:
So RDFXML, N-QUADS, N-TRIPLES, Turtle, etc. all have to use a specific format for blank nodes which means that we can't preserve blank node identifiers with those formats.